cidc_api/config/db.py

```diff
@@ -1,6 +1,6 @@
 from os import environ
 
-from .secrets import get_secrets_manager
```

Comment:
Did these also errored or is it just in case?

Reply:
Just in case.